app/views/finance/participants/_ecf_profile.html.erb

```diff
@@ -72,48 +68,65 @@
 
 <h4>Induction records</h4>
 
-<% pp.induction_records.order(:start_date).each do |ir| %>
+<% pp.induction_records.order(start_date: :desc).each do |ir| %>
   <h5>Induction record: <%= ir.id %></h5>
 
   <%= govuk_summary_list do |summary_list| %>
     <% summary_list.row do |row| %>
       <% row.key { "Training programme" } %>
       <% row.value { ir.induction_programme.training_programme.humanize } %>
+      <% row.action(text: :none) %>
     <% end %>
 
     <% summary_list.row do |row| %>
       <% row.key { "School URN" } %>
       <% row.value { ir.induction_programme.school_cohort.school.urn } %>
+      <% row.action(text: :none) %>
     <% end %>
 
     <% summary_list.row do |row| %>
       <% row.key { "Lead provider" } %>
       <% row.value { ir.induction_programme.partnership&.lead_provider&.name } %>
+      <% row.action(text: :none) %>
     <% end %>
 
     <% summary_list.row do |row| %>
       <% row.key { "Induction status" } %>
       <% row.value { ir.induction_status } %>
+      <% row.action(text: :none) %>
     <% end %>
 
     <% summary_list.row do |row| %>
       <% row.key { "Training status" } %>
       <% row.value { ir.training_status } %>
+      <%
+        if ir == pp.latest_induction_record_for(cpd_lead_provider: ir.cpd_lead_provider)
```

Comment:
how does this work for CIP induction records (they don't have a CPD lead provider). Do we want to have a fallback for those to change the status? 
Also could we move this method into a helper which can be tested?

Reply:
Addressed tks @cwrw